### ISSUE TITLE
Test Claims Decision Requests in staging on LH

### ISSUE THREAD

**Issue by StacyB2023** [department-of-veterans-affairs/va-mobile-app#9809]
## Description 
As part of EVSS Migration, the he Claim Status Tool team (WEB) has completed their rollout for **5103 Waiver- Also known as Claims Decision Requests.**

To stay consistent with WEB, the mobile team will also need to test and switch on the feature flag for this item.

- 

## Acceptance Criteria
  

-  [ ] Test Claims Decision Requests (5103 Waiver) to note the health of the feature.
             **Note: A feature flag is present that can added/removed if needed**

**Comment by StacyB2023** [comment]
@TKDickson I will add this ticket to a thread momentarily for more discussion but just a heads up to check on capacity for your team if any for the coming sprint. Basically this is Prod for WEB and we now need to test on mobile and enable the feature flag. We can do a slow roll out.

cc: @jperk51

**Comment by TKDickson** [comment]
Ultimately testing for identical function with EVSS, and not having re-introduced #6010 from earlier.

Will be creating four claims - 2 8-step and 2 5-step - submitting some evidence (but not all file requests) for all of them, and then requesting decision via EVSS for 1 each of the 8-step and 5-step, and then flipping the feature flag for Lighthouse and requesting decision for the remaining two. That's all data prep. 
The 'test' is then checking the UI and functionality in the mobile app, website, and VBMS systems to make sure that they're all identical for the 5-steps and 8-steps (and across platforms) to the best of my knowledge.

**Comment by TKDickson** [comment]
Found (not related to this development) bugs #9900, #9901, #9902, and confirmed that #5958 is still happening as well (that ticket looked to be super lost, I tried to re-label it to get it back into process).

As demonstrated by the screenshots above, EVSS and Lighthouse handling of 5103s from mobile seem to be doing the same, or equivalent, actions across VBMS. @StacyB2023 @jperk51 QA signs off on switching over to lighthouse for this. Closing ticket.
